payments/vipps/seamless-view.md

```diff
@@ -18,125 +18,427 @@ sidebar:
+    activate Vipps_App
+    note left of Vipps.API: Dialogue with Vipps ⑦
+    Vipps_App-->>Vipps_App: Confirmation Dialogue
+    Vipps_App-->>-Vipps.API: Confirmation
+    activate Vipps.API
+    Vipps.API-->>-SwedbankPay.BackEnd: make payment
+    activate SwedbankPay.BackEnd
+    SwedbankPay.BackEnd-->>SwedbankPay.BackEnd: execute payment
+    SwedbankPay.BackEnd-->>Vipps.API: response
+    SwedbankPay.BackEnd-->>-SwedbankPay.FrontEnd: authorize result
+    activate SwedbankPay.FrontEnd
+    SwedbankPay.FrontEnd-->>-Browser: Display authorize result
+```
+
+### Explainations
+
+* ① When the payer starts the purchase process, you make a `POST` request
+  towards Swedbank Pay with the collected Purchase information.
+* ② `rel: view-payment` is a value in one of the operations, sent as a response
+  from Swedbank Pay to the Merchant.
+* ③ `Open iframe` creates the Swedbank Pay hosted iframe.
+* ④ The consumer UI page displays the payment window as content inside of the
+  `iframe`. The consumer can insert mobile information for authorization.
+* ⑤ A `POST` request is sent to the Vipps API with the mobile number for
+  authorization.
+* ⑥ The response will contain the state of the transaction. It will normally be
+  in `AwaitingActivity` in this fase of the payment.
+* ⑦ Swedbank Pay handles the dialogue with Vipps and the consumer confirms the
+  purchase in the Vipps app.
+
+### 3-D Secure
+
+Swedbank Pay will handle 3-D Secure authentication when this is required. When
+dealing with card payments, 3-D Secure authentication of the cardholder
+is an essential topic. There are two alternative outcome of a card payment:
 
-### API Requests
+1. 3-D Secure enabled - by default, 3-D Secure should be enabled, and Swedbank
+   Pay will check if the card is enrolled with 3-D Secure. This depends on the
+   issuer of the card. If the card is not enrolled with 3-D Secure, no
+   authentication of the cardholder is done.
+2. Card supports 3-D Secure - if the card is enrolled with 3-D Secure, Swedbank
+   Pay will redirect the cardholder to the autentication mechanism that is
+   decided by the issuing bank. Normally this will be done using BankID or
+   Mobile BankID.
 
-The API requests are displayed in the [purchase flow](#purchase-flow).
-The options you can choose from when creating a payment with key `operation`
-set to Value `Purchase` are listed below.
+### Payment Url
+
+{% include payment-url.md when="selecting Vipps as payment instrument" %}
 
-### Options before posting a payment
+## Seamless View Back End
 
-All valid options when posting a payment with operation equal to Purchase,
-are described in [the technical reference][vipps-payments].
+When properly set up in your merchant/webshop site and the payer starts the
+purchase process, you need to make a POST request towards Swedbank Pay with your
+Purchase information. This will generate a payment object with a unique
+`paymentID`. You will receive a **JavaScript source** in response.
```

Comment:
Perhaps explain which operation to look for in order to find the JavaScript source?